### ISSUE TITLE
It seems search function is not working properly

### ISSUE THREAD

**Issue by ryu-s-r** [NoMore201/googleplay-api#118]
Since last week(I remember), searching function returns abnormal data.
here is my debug information below.

```

path = 'https://android.clients.google.com/fdfe/search?c=3&q=Keyword'

data = 
payload {
  searchResponse {
    originalQuery: "Keyword"
    nextPageUrl: "searchList?q=Keyword&o=0&c=3&ksm=1"
  }
}
preFetch {
  url: "searchList?q=Keyword&o=0&c=3&ksm=1"
  response {
    payload {
      listResponse {
        doc {
          backendId: 3
          containerMetadata {
            nextPageUrl: "searchList?q=Keyword&o=0&c=3&ksm=1&ctntkn=CgiK38WpAwIIChAA"
            ordered: true
          }
          unknownCategoryContainer {
          }
        }
      }
    }
  }
  ttl: 7200000
  softTtl: 1800000
}
serverMetadata {
  latencyMillis: 242
}
serverLogsCookie: "\010\022\232\001\027\n\023\010\230\271\263\320\261\234\350\002\025\013\223\226\n\035\030\234\001\227\020\001"
```

I tried fixing it myself but had no luck.
It seems other feature is working well.

**Comment by khaleel-git** [comment]
yes not working properly, it also fetchces unrelated android app ids. I simply filtering unrelated apps first, then showing related apps. 

**How you can achive this:** 
1. first search desired query e.g ("instadp")
2. It will list appIds
3. get details of all appids -> filter out appids with title (if your search query is: "instadp" ignore all appids which does not contain "instadp" word in their title)

**Its a tempoary solution, I also need permanent solution.**

**Comment by 89z** [comment]
FYI this site is not valid:

https://apkfuel.com/apk-downloader/

If you choose x86, you get something else:

~~~
PS C:\> badging -f com.zhiliaoapp.musically.apk
uses-library-not-required:'org.apache.http.legacy'
  uses-feature: name='android.hardware.bluetooth'
  uses-feature: name='android.hardware.location'
native-code: 'armeabi-v7a'
~~~

**Comment by khaleelOrg** [comment]
Its in a development mode. I'm removing the link (I was just elaborating the issue and giving some alternative to fix this temporarily).

**Comment by khaleel-git** [comment]
@89z I am researching about this ([issue#24](https://github.com/89z/googleplay/issues/24#issuecomment-1048792352)), will give u reply once i complete my research.

**Comment by khaleelOrg** [comment]
> badging

You probably got cached version of this link. Please use guest mode or delete cache. I updated the code yesterday & you probably visted before my code update. 

Please see below screenshots.

![image](https://user-images.githubusercontent.com/48614035/155850004-cda958f2-e8e1-4c73-ac5f-ecbeea7c050c.png)

If you serach "badging" on playstore, you got this result (below is the screenshot, no exact app is matched with "badging")

![image](https://user-images.githubusercontent.com/48614035/155850070-d12be30b-66df-477a-b393-8c5bae54f8af.png)

**Comment by khaleel-git** [comment]
> FYI this site is not valid:
> 
> https://apkfuel.com/apk-downloader/
> 
> If you choose x86, you get something else:
> 
> ```
> PS C:\> badging -f com.zhiliaoapp.musically.apk
> uses-library-not-required:'org.apache.http.legacy'
>   uses-feature: name='android.hardware.bluetooth'
>   uses-feature: name='android.hardware.location'
> native-code: 'armeabi-v7a'
> ```

This is in a development mode, native codes not working yet. But Search Suggestion I gave above for filtering the appIds is implemented on this. Please try to search something on it (it won't list unrelated apps).
